src/Components/Checkbox/checkbox.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { useState } from 'react';
+import { TouchableOpacity } from 'react-native';
+
+import Check from '../../../assets/checkbox.svg';
+import Rectangle from '../../../assets/rectangle.svg';
+
+const Checkbox = () => {
```

Comment:
I think a checkbox component might be too granular, as I don't think we're going to have checkboxes in other parts of our app. Perhaps create a `CheckBoxOption` component that is rendered through `FlatList` on the `EligibiliyForm` screen. This could take a callback function as a prop that is triggered when the box is clicked.